tests/queries/0_stateless/03357_block_structure_union_step.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE pk_block_union (d Date DEFAULT '2000-01-01', x DateTime, y UInt64, z UInt64) ENGINE = MergeTree() PARTITION BY d ORDER BY (toStartOfMinute(x), y, z);
```

Comment:
The plan is to test the parallel replicas-enabled case in the dedicated test run?